src/Controller/ControllerFactory.php

```diff
@@ -14,27 +14,33 @@
  * @since         3.3.0
  * @license       https://opensource.org/licenses/mit-license.php MIT License
  */
-namespace Cake\Http;
+namespace Cake\Controller;
```

Comment:
:+1: to the move. Can we include a `class_alias` and deprecation stub for the old name?

Reply:
Done